tests/testsrc/jstests/harmony/typed-array-ctor.js

```diff
@@ -20,15 +20,15 @@ load("testsrc/assert.js");
 
   msg = null;
   try { new Int8Array(new ArrayBuffer(), -1); } catch (e) { msg = e.toString();}
-  assertEquals("RangeError: offset -1 out of range", msg);
+  assertEquals("RangeError: index out of range", msg);
```

Comment:
this msg text change is fine for me, but personally i really like to have the wrong value of the index in the errror message (yes ff does not do that but...)

I like to have something like
* RangeError: index '-1' out of range
* RangeError: index value -1 out of range

Reply:
FF:  RangeError: invalid or out-of-range index
Chrome: RangeError: Invalid typed array length: -1

Reply:
Looks like it doesn't. Just pushed these changes.